packages/terra-clinical-result-view/src/ResultView.scss

```diff
@@ -26,7 +26,7 @@
   }
 
   .result-view-body--padded {
-    padding: var(--result-view-body-padding, 0.3125rem);
+    padding: var(--result-view-body-padding, 0.3125em);
```

Comment:
How come we switched to `em`?

Reply:
This component can be consumed in different places with different font sizes. ems would be a better choice like @neilpfeiffer suggested.